test/attr/attr_objc.swift

```diff
@@ -1121,6 +1122,7 @@ class infer_instanceVar1 {
 // CHECK-LABEL: @objc var var_Optional12: OpaquePointer?
 // CHECK-LABEL: @objc var var_Optional13: UnsafeMutablePointer<Int>?
 // CHECK-LABEL: @objc var var_Optional14: UnsafeMutablePointer<Class_ObjC1>?
+// CHECK-LABEL: @objc var var_Optional15: NSRange?
```

Comment:
…and now I take it back, because we don't actually bridge them _correctly._ Will fix.